### DISCUSSION TITLE
Wildcardsearch

### DISCUSSION THREAD

**Discussion by jume-dev** [quickwit-oss/quickwit#346] [Q&A] [answered]
Hi,
just a quick question, is it possible to do a wild card search on a text field? I have a simple test data structure of people with firstname, lastname and address. Firstname is mostly only one word and I wanted to search on word fragments, like for example if I had a Willy in my data, I wanted to search for illy and find him.
Is this possible yet and if so, how?
Best regards

**Comment by fmassot** [comment, marked as answer]
Hi @jume-dev 
I think what you are searching for is the [Ngram](https://docs.rs/tantivy/0.15.3/tantivy/tokenizer/struct.NgramTokenizer.html) tokenizer. It is not yet available in Quickwit but it should be here soon as we want to make tantivy tokenizers available in Quickwit.
In the meantime, you can try using tantivy, [tantivy-cli](https://github.com/tantivy-search/tantivy-cli) is a good start.

**Reply by jume-dev** [reply]
Alright, thanks. I'm looking forward to trying that out once you have it ready :)
